### ISSUE TITLE
Unable to specify pino options

### ISSUE THREAD

**Issue by Heath101** [felixheck/laabr#62]
Thanks for this great project.  When using v6.0.2 of laabr, I am able to specify level option for pino.
```javascript
{
  plugin: require( "laabr" ),
  options: {
    pino: {
      level: "debug"
    }
  }
}
```
but when using v6.1.0 I get a validation error

```
Error [ValidationError]: "pino.level" is not allowed
```
Is this a regression or intended to be more restrictive? or am I missing something?  Here is the commit that seems to have removed the functionality.  [https://github.com/felixheck/laabr/commit/d3e9ae470921dc85ed31e4e3b2138c0bb349d4d1](https://github.com/felixheck/laabr/commit/d3e9ae470921dc85ed31e4e3b2138c0bb349d4d1)

We need to be able to specify the log level so that we can silence the logging that occurs when running our integration tests.

**Comment by felixheck** [comment]
Hiya — thanks for reaching out to me and sorry for the caused inconvenience.
It's not intended at all. The commit you mentioned should be even less restrictive and allow all kind of options; I forgot to allow unknown keys though. I just released `6.1.1` — would you mind checking it? 😊

**Comment by Heath101** [comment]
No worries.  Worked like a charm.  Thanks!
